php_laravel/app/Http/Controllers/ChargeController.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+use Illuminate\Http\RedirectResponse;
+use OmiseCharge;
+use OmiseSource;
+use OmiseToken;
+use Illuminate\Support\Facades\Http;
+use App\Models\Payment;
+
+class ChargeController extends Controller
+{
+    //
+    public function index() : View {
+        return view('formcharge');
+    }
+
+    /**
+     * 基本的にchargeをするときは、tokenを作って、それを元にchargeを作るものと理解してる
+     */
```

Comment:
Could you please translate the comment into English?

Reply:
Sure, sorry.

Reply:
@aashishgurung 
Done
https://github.com/omise/examples/pull/29/commits/64d875dae2316e04d267dd2750d5c4c4f75ceae5